pg-auto-failover.spec

```diff
@@ -60,5 +60,10 @@ PATH=%{pginstdir}/bin:$PATH
 
 
 %changelog
+* Mon May 3 2019 - Nils Dijk <nils@citusdata.com> 1.0.1
```

Comment:
I think that one should also be a single entry such as:
  - Official release for 1.0.1